### PR TITLE
Remove turbo

### PR DESCRIPTION
**Pull request by danielpassos**

### CODE REVIEW

**Comment by jhellar** [comment]
reviewing

File: README.md [hunk 1]
```diff
@@ -1,27 +1,46 @@
...
+```json
+{
+}
+```
 
-# Google APIs [/cloud/googleapis]
+## Tests
 
-Shorten url
+All the tests are in the "test/" directory. The cloud app is using mock as the test runner.
```
Comment (jhellar) [inline README.md]: should be mocha instead of mock

**Comment by jhellar** [comment]
LGTM, just one comment. Verified that `grunt serve`, `npm test` and `npm coverage` works.

**Comment by danielpassos** [comment]
@jhellar Fixed!

**Pull request merged**